### PR TITLE
Updates start silent, turning on startWithAudioMuted and few UI tweaks.

### PR DESCRIPTION
**Pull request by damencho**
Disabled mic unmute button and removes remote participants volume slider.

### CODE REVIEW

File: css/_toolbars.scss [hunk 1]
```diff
@@ -116,9 +116,9 @@
             }
         }
         i.disabled, .disabled i {
-            cursor: initial;
-            color: #fff;
-            background-color: #a4b8d1;
+            cursor: initial !important;;
+            color: #fff !important;;
+            background-color: #a4b8d1 !important;;
```
Comment (virtuacoplenny) [inline css/_toolbars.scss]: Why do you need !important? What styling is taking higher priority?

Comment (damencho) [reply css/_toolbars.scss]: Yep it is .icon-mic-disabled 
image

Maybe that one should be removed ...

Comment (damencho) [reply css/_toolbars.scss]: It seems to me that icon-mic-disabled is for muted ... and it always overrides the disabled css styling.

**Comment by damencho** [comment]
Depends on https://github.com/jitsi/lib-jitsi-meet/pull/912 to be complete.

**Pull request merged**